convert2rhel/applock.py

```diff
@@ -103,8 +101,28 @@ def _try_create(self):
 
```

Comment:
Note: the `raise exc` on line  100/98 above would be better as `raise` without the exception object.  On Python3 this won't make a difference but on Python2, the original traceback object is used with `raise` but a new traceback object is generated with `raise exc`, losing the original.